docs/networks/mainnet.md

```diff
@@ -0,0 +1,104 @@
+# Mainnet Configuration
+
+The Graph Network's testnet is on Ethereum, and network information can be found at https://thegraph.com/explorer/network.
+
+## Latest Releases
+
+| Component       | Release                                                                    |
+| --------------- | -------------------------------------------------------------------------- |
+| contracts       | [1.11.1](https://github.com/graphprotocol/contracts/releases/tag/v1.11.1)  x|
+| indexer-agent   | [0.18.6](https://github.com/graphprotocol/indexer/releases/tag/v0.18.6)    |
+| indexer-cli     | [0.18.6](https://github.com/graphprotocol/indexer/releases/tag/v0.18.6)    |
+| indexer-service | [0.18.6](https://github.com/graphprotocol/indexer/releases/tag/v0.18.6)    |
+| graph-node      | [0.28.2](https://github.com/graphprotocol/graph-node/releases/tag/v0.28.2) |
+
+## Registration / Funding (GRT)
+
+The Graph Network mainnet is open for everyone to participate in as an
+indexer. The only requirement is a minimum stake of 100k GRT.
+
+## Network Parameters
```

Comment:
It'd be nice to remove this section, but the epoch length currently isn't displayed at https://thegraph.com/explorer/network. 

Making note of it.

Reply:
👍